weejson/src/main/scala-3/com/rallyhealth/weejson/v1/package.scala

```diff
@@ -0,0 +1,6 @@
+package com.rallyhealth.weejson
+
+package object v1 {
+  private val CollectionConverters = scala.jdk.CollectionConverters
```

Comment:
A few months back I integrated a locally-published version of this PR with quite a bit of client code, both Scala 2 and 3, and did not see any collisions. I've validated that a lot of that code has wildcard imports exactly as you describe. So I don't believe this will be an issue.